src/main/kotlin/frc/robot/subsystems/superstructure/climb/ClimbSubsystem.kt

```diff
@@ -0,0 +1,52 @@
+package frc.robot.subsystems.superstructure.climb
+import com.revrobotics.CANSparkMaxLowLevel
+import org.ghrobotics.lib.commands.FalconCommand
+import org.ghrobotics.lib.mathematics.units.*
+import org.ghrobotics.lib.mathematics.units.nativeunit.*
+import org.ghrobotics.lib.motors.rev.FalconMAX
+import org.ghrobotics.lib.wrappers.FalconDoubleSolenoid
+import org.ghrobotics.lib.wrappers.FalconSolenoid
+
+object ClimbSubsystem {
+
+    class Climb(val climbHeight : SIUnit<Meter>): FalconCommand() {
+
+        //TODO Tune PID / Enter Values
+        val Kp = 0.00
+        val Kd = 0.00
+        val solenoid = FalconDoubleSolenoid(0, 1, 8)
+        //This code was wrote assuming we are useing a "yeet stick" that pulls us up the poll TODO Tune this!!
+        val climbMaster: FalconMAX<Meter> = FalconMAX(21, CANSparkMaxLowLevel.MotorType.kBrushless, NativeUnitLengthModel(4096.nativeUnits, 0.75.inches))
+        val climbSlave = FalconMAX(1   /*CHANGE PORT*/, CANSparkMaxLowLevel.MotorType.kBrushless, DefaultNativeUnitModel)
```

Comment:
Same with these, needs to be defined within the scope of climbsubsystem rather than the command